.gitlab-ci.yml

```diff
@@ -55,7 +55,8 @@ noether-cpu:
     - make -k -j$NPROC_CPU BACKENDS="$BACKENDS_CPU" JUNIT_BATCH="cpu" junit search=nek NEK5K_DIR=$NEK5K_DIR
 # Clang-tidy
     - echo "-------------- clang-tidy ----------" && clang-tidy --version
-    - TIDY_OPTS="-fix-errors" make -j$NPROC_CPU tidy && git diff --color=always --exit-code
+    - export OCCA_DIR= && make lib -j$NPROC_CPU
+    - make -j$NPROC_CPU tidy
```

Comment:
Will this still show what code needs to be changed and a suggest for how to change it?

Reply:
Yeah, it should